backend/models/ticket.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const ticketSchema = new mongoose.Schema(
+  {
+    flightNumber: { type: String, required: true },
+    flightId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Flight',
+      required: true,
+    },
+    userId: {
+      type: mongoose.Schema.Types.ObjectId,
```

Comment:
Could you use string instead of mongoose.Schema.Types.ObjectId for external ids?